generators/app/templates/service/src/utils/query.ts

```diff
@@ -1,15 +1,15 @@
-import { QueryOptions } from '@cxcloud/ct-types/common';
+import { QueryOptions } from '@cxcloud/ct-types/common'; 
 
 export function getQueryOptions(
-	page?: number,
-	perPage?: number,
-	sortPath?: string,
-	ascending?: boolean
+  page?: number,
+  perPage?: number,
+  sortPath?: string,
+  ascending?: boolean
 ): QueryOptions {
-	return {
-		page: typeof page !== 'number' || page < 1 ? 1 : page,
-		perPage: typeof perPage !== 'number' || perPage < 1 ? 20 : perPage,
-		sortPath,
-		ascending
-	};
+  return {
+	page: typeof page !== 'number' || page < 1 ? 1 : page,
```

Comment:
Sorry Linda but this is still not fixed. did you install Prettier? Also install "EditorConfig" to make it easier.